TLSharp.Core/Types/ParticipantTypes.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace TLSharp.Core.Types
+{
+    public enum ParticipantTypes
```

Comment:
Change this to something like ParticipantTypesFilter too maybe?

Reply:
ParticipantTypeFilters

Reply:
ok